test/topology/test_topology.py

```diff
@@ -45,6 +45,8 @@ async def test_remove_server_add_column(manager, random_tables):
     servers = await manager.servers()
     table = await random_tables.add_table(ncolumns=5)
     await manager.server_add()
+    server_uuid = await manager.get_server_uuid(servers[1])   # get uuid [1]
+    await manager.remove_node(servers[1], server_uuid)        # Remove   [1]
```

Comment:
`remove_node` should take two parameters: the IP of the removing server and the Host ID of the server being removed.

Reply:
Manual initiator IP set in https://github.com/scylladb/scylladb/commit/7cc3aedb8d083a86de5cf4d92fd4c3190af4413d